source/MRCuda/MRCudaFastWindingNumber.cu

```diff
@@ -64,6 +64,78 @@ namespace Cuda
         }
     }
 
+    __device__ float calcDistance( const float3& pt, const Node3* nodes, const float3* meshPoints, const FaceToThreeVerts* faces, float maxDistSq, float minDistSq )
```

Comment:
is this function duplicated from MRMeshProjection.cu?

Reply:
No, they are not completely the same